enos/modules/worker/templates/worker.hcl

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+listener "tcp" {
+  purpose = "proxy"
+  tls_disable = true
+  address = "0.0.0.0"
+}
+
+worker {
+  # Name attr must be unique across workers
+  name = "worker-${id}"
+  description = "Enos Boundary worker ${id}"
+
+  # Workers must be able to reach controllers on :9201
+  initial_upstreams = ${controller_addresses}
+
+  public_addr = "${public_addr}"
+
+  tags {
+    region = ["${region}"]
+    type = ${type}
+  }
+}
+
+# must be same key as used on controller config
+kms "awskms" {
+  purpose    = "worker-auth"
+  region     = "${region}"
+  kms_key_id = "${kms_key_id}"
+}
```

Comment:
I'm still a little confused about the pki worker type we're seeing in boundary. I did notice that this configuration file looks different than what's in the docs?

https://developer.hashicorp.com/boundary/docs/configuration/worker/kms-worker